<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2010 - 2012 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 - 2009 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>



 <comment> All rights reserved. </comment>
 <class>CQMiriamWidget</class>
 <widget class="CopasiWidget" name="CQMiriamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>456</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>MIRIAM Annotation</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mpLblCreated">
     <property name="text">
      <string>Created at</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="mpSpcCreated">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Double click on the number of Reference / Description to view.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnClear">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mpLblModified">
     <property name="text">
      <string>Modified at</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mpLblAuthors">
     <property name="text">
      <string>Authors</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mpLblReferences">
     <property name="text">
      <string>References</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mpLblDescription">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="CQTableView" name="mpTblDescription">
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="CQTableView" name="mpTblModified">
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CQTableView" name="mpTblReferences">
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="CQTableView" name="mpTblAuthors">
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateTimeEdit" name="mpDTCreated"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQTableView</class>
   <extends>QTableView</extends>
   <header>copasi/UI/CQTableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpDTCreated</tabstop>
  <tabstop>mpTblAuthors</tabstop>
  <tabstop>mpTblReferences</tabstop>
  <tabstop>mpTblDescription</tabstop>
  <tabstop>mpTblModified</tabstop>
  <tabstop>mpBtnDelete</tabstop>
  <tabstop>mpBtnClear</tabstop>
 </tabstops>
 <includes>
  <include location="global">vector</include>
  <include location="local">copasi/UI/copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnClear</sender>
   <signal>pressed()</signal>
   <receiver>CQMiriamWidget</receiver>
   <slot>slotBtnClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>pressed()</signal>
   <receiver>CQMiriamWidget</receiver>
   <slot>slotBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpDTCreated</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>CQMiriamWidget</receiver>
   <slot>slotCreatedDTChanged(QDateTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
